plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -328,6 +330,20 @@ func (f *Fetcher) getOrgs(ctx context.Context, uID string) ([]map[string]interfa
 	return results, nil
 }
 
+func formatObject(org any) (map[string]any, error) {
```

Comment:
You can be more explicit about what the function does and name it something like `toJSONMap`.